index.html

```diff
@@ -2107,6 +2121,105 @@ <h3>Securing Verifiable Credentials</h3>
 Directory [[VC-SPECS]].
         </p>
 
+        <section class="normative">
+          <h4>The Verification Algorithm</h4>
+
+          <p>
+This section contains an algorithm that <a>conforming verifier processors</a>
+can run when verifying a <a>verifiable credential</a> or a <a>verifiable
+presentation</a>. This algorithm takes a byte sequence (bytes
+<var>inputBytes</var>) and a media type (string <var>inputMediaType</var>) as
+inputs, and produces an object (<var>verified</var>) that contains a
+verification status (boolean <var>verified</var>.<var>status</var>), a verified
+document (object <var>verified</var>.<var>document</var>), a media type (string
+<var>verified</var>.<var>mediaType</var>), zero or more optional warnings (array
+of <a>anomalyDescriptions</a> <var>verified</var>.<var>warnings</var>), and zero
+or more optional errors (array of <a>anomalyDescriptions</a>
+<var>verified</var>.<var>errors</var>).
```

Comment:
This comment is purely editorial, and may be postponed for later.

I wonder whether we should not explicitly rely on the [infra standard](https://infra.spec.whatwg.org/) by putting in references to terms like ["byte sequence"](https://infra.spec.whatwg.org/#byte-sequences), ["string"](https://infra.spec.whatwg.org/#strings), use the concept of ["list"](https://infra.spec.whatwg.org/#lists) instead of arrays, and ["maps"](https://infra.spec.whatwg.org/#maps) instead of objects. This may be a bit of a pain because probably we should then use someting like _verified["warnings"]_ instead of _verified.warnings,_ but it would avoid starting discussions about, say, Unicode code points for the incoming documents (because that is amply detailed in the infra standard for strings).

Just a thought.

Reply:
Yeah, I'm delaying pulling infra in for as long as possible, it caused a lot of problems when we started to use it in DID Core that I'd like to avoid. If people can't read "array" and "object" and know what we're talking about, then I don't think Infra is going to help those implementers. :)

In any case, I hear you, we can pull Infra in when somebody starts demanding that our algorithms have that level of precision, but honestly, these algorithms are not that complex and don't need to be typed so strictly.